notebooks/classical_training.ipynb

```diff
@@ -431,9 +549,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.10"
+   "version": "3.10.4"
```

Comment:
Was the version change needed?

Reply:
Ah, this was auto-generated by Jupyter Lab in the metadata section of the notebook. Yeah we can keep this as-is.